apollo-compiler/src/main/kotlin/com/apollographql/apollo/compiler/ast/AST.kt

```diff
@@ -92,6 +92,11 @@ internal data class ObjectType(
   }
 }
 
+internal class FragmentType(
+    val objectType: ObjectType,
+    val packageName: String
+)
```

Comment:
Can you add `packageName: String?` to `ObjectType` instead? It doesn't really make sense to have an AST type named "Fragment" as Kotlin doesn't know anything about a Fragment.

Reply:
Mmm but there's also an OperationType too so maybe it doesn't matter that much. @sav007 any opinion about this ?

Reply:
I think your point that it doesn't map to a `Kotlin` concept is valid. The reason I didn't place it on `ObjectType` is that `ObjectType` is also used as an intermediary object in a couple of places so it feels like it is breaking this principle already. I used a separate type to more closely tie package name to class generation logic which allows the type system to ensure that packageName is provided. Let me know if you want me to change that, not a big deal to me just wanted to explain my position.